src/jobflow_remote/cli/project.py

```diff
@@ -387,3 +391,114 @@ def list_worker(
     project = cm.get_project()
     table = get_worker_table(project.workers, verbosity)
     out_console.print(table)
+
+
+#####################################
+# Edit app
+#####################################
+
+
+app_edit = JFRTyper(
+    name="edit",
+    help="Edit the project files",
+    no_args_is_help=True,
+)
+app_project.add_typer(app_edit)
+
+
+@app_edit.command()
+def replace(
+    old_string: Annotated[str, typer.Argument(help="String to search for and replace")],
+    new_string: Annotated[str, typer.Argument(help="String to replace with")],
+    all_projects: Annotated[
+        bool,
+        typer.Option(
+            "--all",
+            "-a",
+            help="Apply replacement to all project files",
+        ),
+    ] = False,
+    force: force_opt = False,
```

Comment:
Probably to be changed to always_confirm / --yes / -y when we do that PR (or would you start this option directly with --yes ?)